docs/reference/remap/functions/redact.cue

```diff
@@ -0,0 +1,94 @@
+package metadata
+
+remap: functions: redact: {
+	category: "String"
+	description: """
+		Redact potentially sensitive data in `value` such as:
+
+		- credit card numbers
+		- custom patterns
+		- (more to come!)
+
+		This can act as a last resort in the case your applications log sensitive data by accident to ensure you do not
+		leak this information into your log storage.
+		"""
+
+	arguments: [
+		{
+			name: "value"
+			description: #"""
+				The value to redact sensitive data from.
+
+				Its behavior differs depending on the type of `value`:
+				- For strings, it simply redacts the sensitive data and returns a new string
+				- For arrays, it redacts the sensitive data in each string element
+				- For objects, it masks the sensitive data in each string value, but not keys
+
+				For arrays and objects it will recurse into any nested arrays or objects. Any non-string elements will
+				be skipped.
+
+				Any redacted text will be replaced with `[REDACTED]`.
+				"""#
+			required: true
+			type: ["string", "object", "array"]
+		},
+		{
+			name: "filters"
+			description: #"""
+					List of filters to be applied to the `value`.
+
+					If none, the default set of filters is applied: ["credit_card"].
```

Comment:
Don't we have a `default` attribute that specifies the default value?

Reply:
Ah yes 👍